packages/amplication-client/src/Purchase/PurchasePage.tsx

```diff
@@ -119,9 +120,12 @@ const PurchasePage = (props) => {
     },
     [upgradeToPro, handleContactUsClick]
   );
-
+  const pageTitle = "Pricing & Plans";
   return (
     <Modal open fullScreen>
+      <Helmet>
+        <title>{`Amplication${pageTitle ? ` | ${pageTitle}` : ""}`}</title>
```

Comment:
Page title is const, the is no need for validation.

Reply:
Oh sure thing ! I will revert other changes, remove the validation and commit